server/router/src/test/java/org/infinispan/server/router/utils/HotRodClientTestingUtil.java

```diff
@@ -0,0 +1,52 @@
+package org.infinispan.server.router.utils;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+
+import java.net.InetAddress;
+
+public enum HotRodClientTestingUtil {
```

Comment:
Granted but all Hot Rod tests require connecting to them somehow... I could write some custom client only for testing this module but chances to make buggy implementation are pretty high whereas we already build a nice Hot Rod client... 

Moving the tests to the client seems also a bit weird... we are testing server functionality so why they are in the client?

It's a bit chicken/egg problem with no good solution. I would prefer to leave it as is (unless anyone has good arguments why I should not do it).